CHANGES.txt

```diff
@@ -153,7 +153,7 @@ Coming in build 307, as yet unreleased
   * Removed redundant and obsolete references to older python unicode compatibility (#2085, @Avasam)
   * Use byte-string (`b""`) for constant bytes values instead of superfluous `.encode` calls (#2046, @Avasam)
   * Cleaned up unused imports (#1986, #2051, #1990, #2124, #2126, @Avasam)
-  * Removed duplicated declarations, constants and definitions (#2050 , #1950, #1990, @Avasam)
```

Comment:
probably not worth a new entry for this.

Reply:
Especially that far back ^^" It's such a tiny optimization, I agree